Adding StarCraft to WinePak

Isakku opened the pull request:
Changed icon and  scripts, testing is required.
Followed @Saroufim instructions here
https://github.com/winepak/applications/issues/116#issuecomment-412320014
I'm no expert with Wine or Flatpak, this is my first pull request here.

Isakku committed:
Adding StarCraft to WinePak

Changed icon and  scripts, testing is required.

LeandroStanger commented:
Add the correct icon https://us.shop.battle.net/static/3.2.1/images/family-icons/starcraft-remastered.svg

Isakku commented:
> Add the correct icon https://us.shop.battle.net/static/3.2.1/images/family-icons/starcraft-remastered.svg 

Lol, and that's the icon we are supposed to use? I really liked the Terran Wraith that SC1 had and I loved that new Wraith redesign they supplied with 1.16, but now they give us this bland SC letters... *sigh* OK, I'm doing it.

EDIT
Hmm. while we are at it, could you please tell me how do you test this? I mean, how do you test winepak scripts that are offline and were made by yourself with mostly possible horrific bugs  without breaking your system (I already found a bug with SC2 winepak) https://github.com/winepak/applications/issues/122
When I was editing it to make the SC1 script I noticed that it changes Wine graphics mode to opengl... I don't know why. Lutris SC2 install script does many other different things but nothing like that.

Isakku committed:
Uploaded correct icon

Lol, and that's the icon we are supposed to use? I really liked the Terran Wraith that SC1 had and I loved that new Wraith redesign they supplied with 1.16, but now they give us this bland SC letters... sigh OK, I'm doing it.

LeandroStanger commented on com.blizzard.StarCraft/com.blizzard.StarCraft.appdata.xml:
```diff
@@ -0,0 +1,90 @@
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>LicenseRef-proprietary</project_license>
+    <name>StarCraft</name>
+    <summary>The Ultimate Real-time Strategy Game</summary>
```
```
<summary>(RTS) Estrategia en tiempo real</summary>

Isakku replied:
I took that text from official website. I don't see the need to change it.

LeandroStanger commented on com.blizzard.StarCraft/com.blizzard.StarCraft.appdata.xml:
```diff
@@ -0,0 +1,90 @@
+    <summary>The Ultimate Real-time Strategy Game</summary>
+    <summary xml:lang="pt_BR">Estratégia em tempo real (RTS)</summary>
+    <summary xml:lang="es">(RTS) Estrategia en tiempo real</summary>
+    <description><p>Infinite fronts, infinite enemies. Real-time Strategy Restored. Command the mechanized Terrans, psi-powered Protoss, and insectoid Zerg as they vie for map control of eight unique environments. Build your base and conscript your army in a real-time, military sci-fi vision of the future. Once you’ve finished the campaign, go head-to-head or team up with other players online. Pi ...
```
```
<description><p>StarCraft introdujo muchas características revolucionarias que se han convertido en elementos básicos del género de la estrategia en tiempo real: facciones asimétricas, equilibrio meticuloso y un fuerte énfasis en la estrategia y la accesibilidad. Descubre por qué, después de más de quince años, miles de jugadores de todo el mundo aún siguen disfrutando de la frenética acción de StarCraft.</p></description>

Isakku replied:
Again this is  from official website. I don't see the need to change it.

LeandroStanger commented on com.blizzard.StarCraft/com.blizzard.StarCraft.appdata.xml:
```diff
@@ -0,0 +1,90 @@
+    <summary xml:lang="pt_BR">Estratégia em tempo real (RTS)</summary>
+    <summary xml:lang="es">(RTS) Estrategia en tiempo real</summary>
+    <description><p>Infinite fronts, infinite enemies. Real-time Strategy Restored. Command the mechanized Terrans, psi-powered Protoss, and insectoid Zerg as they vie for map control of eight unique environments. Build your base and conscript your army in a real-time, military sci-fi vision of the future. Once you’ve finished the campaign, go head-to-head or team up with other players online. Pi ...
+    <description xml:lang="pt_BR"><p>Batalhas infinitas, inimigos infinitos.Estratégia em tempo real revitalizada. Lidere os terranos com suas máquinas mortíferas, os protoss de poderes psiônicos, e os zergs insectiformes em batalhas territoriais por oito cenários únicos. Mergulhe num futuro militarista de ficção-científica. Construa uma base e aliste o seu exército. Tudo isto numa experiência em ...
```
```
<description xml:lang="pt_BR"><p>StarCraft introduziu muitos recursos revolucionários que desde então se tornaram padrão no gênero de estratégia em tempo real: facções assimétricas, equilíbrio meticuloso e forte ênfase em estratégia aprofundada e alta acessibilidade. Descubra por quê, após mais de dez anos, StarCraft continua a fazer sucesso em torneios no mundo inteiro.</p></description>

Isakku replied:
Yet again, the text is from official website.

LeandroStanger commented on com.blizzard.StarCraft/com.blizzard.StarCraft.appdata.xml:
```diff
@@ -0,0 +1,90 @@
+    <summary xml:lang="es">(RTS) Estrategia en tiempo real</summary>
+    <description><p>Infinite fronts, infinite enemies. Real-time Strategy Restored. Command the mechanized Terrans, psi-powered Protoss, and insectoid Zerg as they vie for map control of eight unique environments. Build your base and conscript your army in a real-time, military sci-fi vision of the future. Once you’ve finished the campaign, go head-to-head or team up with other players online. Pi ...
+    <description xml:lang="pt_BR"><p>Batalhas infinitas, inimigos infinitos.Estratégia em tempo real revitalizada. Lidere os terranos com suas máquinas mortíferas, os protoss de poderes psiônicos, e os zergs insectiformes em batalhas territoriais por oito cenários únicos. Mergulhe num futuro militarista de ficção-científica. Construa uma base e aliste o seu exército. Tudo isto numa experiência em ...
+    <description xml:lang="es"><p>Infinidad de frentes, infinidad de enemigos Estrategia en tiempo real renovada. Dirige a los terran mecanizados, a los protoss con energía psiónica y a los zerg insectoides en su lucha por obtener el control del mapa en ocho entornos únicos. Construye tu base y recluta a tu ejército en tiempo real en esta versión militarizada de un futuro de ciencia ficción. Cuan ...
```
```
<description xml:lang="es"><p>StarCraft introdujo muchas características revolucionarias que se han convertido en elementos básicos del género de la estrategia en tiempo real: facciones asimétricas, equilibrio meticuloso y un fuerte énfasis en la estrategia y la accesibilidad. Descubre por qué, después de más de quince años, miles de jugadores de todo el mundo aún siguen disfrutando de la frenética acción de StarCraft.</p></description>

Isakku replied:
*SIGH* I don't see a point in these changes. the text comes from the official website.

Isakku commented:
There is really no way I can make this work if the code It is based on (StarCraft II Winepak) fails on my side (issue #122)
